<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramesWidget</class>
 <widget class="QWidget" name="FramesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TypeLabel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableWidget" name="frameSequenceListTableWidget">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>First</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>Video 2D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left-eye video</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right-eye video</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>frameSequenceListTableWidget</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>FramesWidget</receiver>
   <slot>refreshFrames()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FramesWidget</receiver>
   <slot>refreshFrames()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
